aws/rust-runtime/aws-credential-types/src/credentials_impl.rs

```diff
@@ -164,12 +179,102 @@ impl Credentials {
         &mut Arc::make_mut(&mut self.0).expires_after
     }
 
+    /// Returns the account ID.
+    pub fn account_id(&self) -> Option<&AccountId> {
+        self.0.account_id.as_ref()
+    }
+
     /// Returns the session token.
     pub fn session_token(&self) -> Option<&str> {
         self.0.session_token.as_deref()
     }
 }
 
+/// Builder for [`Credentials`]
+///
+/// Similar to [`Credentials::new`], the use of the builder is intended for a custom credentials provider implementation.
+/// It is __NOT__ secure to hardcode credentials into your application.
+#[derive(Default, Debug, Clone)]
```

Comment:
Should this be deriving `Debug`? We don't for `Credentials` it seems and presumably so we don't leak credentials in logs accidentally.

Reply:
Thanks for catching. Removed it in b93ec3a.